BP/common/religion/_ToDo.info

```diff
@@ -0,0 +1,60 @@
+To Do Tasks:
+Fix some remaining errors:
+	[12:18:33][jomini_dynamicdescription.cpp:66]: Unrecognized loc key special_doctrine_christian_ecumenism_name. special_doctrine_christian_ecumenism
+
+Remove Nuristani relgion after merge
+
+==================
+Eventual work:
+------------------
+Big Tasks
+~~~~~~~~~~~~~~~~~~
+* Revisit Christian doctrine and tenet interactions
+	Demigurge: Negate Opinion Bump for Jews with Christian Syncretism (will require more work on understanding how to do triggered opinion mechanics)
+* See if new category besides "shunned" is possible for antinomians (shunned effects w/o hooks or devotion loss, just opinion loss)
+	Yes, but touches quite a bit of code for a more precise flavorization and opinion modeling...
+* More interaction between Heterodox Hindu Philosophies and options:
+	E.g., no determining a Bhakti or founding holy orders or pilgrimages
+	Figure out how to dynamically remove opinion penalties for criminal/shunned activites
+* Revisit Christian doctrine and tenet interactions
+	Demigurge: No wandering Jews in your court
+	Judaizers: Increased chances for Jews in your court
+
+~~~~~~~~~~~~~~~~~~
+Moderate Tasks
+~~~~~~~~~~~~~~~~~~
+* Replace peregrinition_decision select_and_move_to_pool_effect with new effect that makes pagan locales more popular
+* Add baldness/tonsure to peregrinition tenet for monks
+* Make it so other pagans can adopt sacred eunuchs
+* Implement some sort of Bhakti innovations for Dravidians
+* Monument building decision like raising a runestone for Germanics
+	Needs to interact wtih fp1 it seems!
+
+~~~~~~~~~~~~~~~~~~
+Little Tasks
+~~~~~~~~~~~~~~~~~~
+* Improve interaction between Communion tenet and Eucharistic Theology
+* Implement cultural localization for Kalash/Nuristani Hinduism
+* Add events related to Tantric for Esoteric Buddhists/Hindus
+* Add more gameplay interaction between tenets and Christian doctrines
+
+==================
+General Catholic/Orthodox Virtues:
+Chastity: All Christians
+Temperance: Monasticism
+Charity/Generosity: 
+Diligence: 
+Patience: 
+Gratitude/Kindness: All Christians (Compassionate)
+Humility: Original Sin
+==================
+
+Following WtWSMS faiths are still not included:
+Eiateuta (Illyrian Pagan), Hubalism, Wodenism
```

Comment:
Wodenism and Hubalism aren't faiths used in WtWSMS and you have imported Eiateuta. With this in mind the import is finished.